### DISCUSSION TITLE
project scope discussion

### DISCUSSION THREAD

**Discussion by travisghansen** [routernetes/routernetes#7] [General]
I'd love to discuss this project a little deeper with you (I sent a request on discord since that was the only communication channel available on your github profile).

**Comment by loganmc10** [comment]
Ah understood, can you resend the request, or just try sending me a DM? I normally just ignore unknown friend requests but I think (I could be totally wrong) that you can just send me a DM on there.

**Reply by travisghansen** [reply]
ok, tried both..
